compiler/next/tools/chpldoc/chpldoc.cpp

```diff
@@ -18,117 +18,751 @@
  * limitations under the License.
  */
 
+/*
+  To test this today, you can force sub_test to use this version by doing:
+  > CHPL_CHPLDOC_NEXT=$(find $(pwd) -name chpldoc -type f) ./util/test/start_test.py test/chpldoc/enum.doc.chpl
+  You'll want to make sure that the find command finds a single chpldoc executable
```

Comment:
An alternative would be to adjust `compiler/main/driver` to run this alternative chpldoc implementation.